.changeset/dirty-socks-sip.md

```diff
@@ -0,0 +1,32 @@
+---
+'@astrojs/markdown-remark': minor
+'astro': minor
+---
+
+Adds a configuration called https://shiki.style/guide/load-lang#custom-language-aliases, that allows a non-supported code language to a known language.
+ 
+The below example will tell shiki to highlight the code blocks `cjs` using the `javascript` syntax highlighting.  
```

Comment:
```suggestion
Adds a new Shiki configuration option `langAlias` to use [Shiki's `langAlias` option](https://shiki.style/guide/load-lang#custom-language-aliases). This allows you to define custom aliases for [Shiki's bundled language IDs](https://shiki.style/languages#bundled-languages) (such as the built-in `js` for `javascript`) for Markdown code blocks.

This can be used to map a non-supported code language to a known language, allowing you to indicate the exact language in your source code while using appropriate highlighting.
 
The example below configures any code blocks using the language alias `cjs` to use `javascript` syntax highlighting.  
```
Maybe something like this? I took a look at the original feature, and their example seems a little more general purpose than only using a language that doesn't have a supported code. It seems you could create an alias for any reason, using anything, so I think describing this as one particular use case vs. "what this thing is for" frames it as being able to be used in other ways, too.

Reply:
@sarah11918 I updated the changeset based on https://github.com/withastro/astro/pull/12039#discussion_r1768629167, can you provide a new suggestion?